src/frontend/src/components/public/PIAFormTabs/review/ProgamArea/displayProgramArea.tsx

```diff
@@ -37,8 +37,11 @@ const DisplayProgramArea = (props: IDisplayProgramAreaProps) => {
   const canEditProgramAreaReview =
     reviewPagePrivilegeParams?.editProgramAreaReview ?? false;
 
-  const allowUserReviewProgramArea = () => {
-    if (!canEditProgramAreaReview) return false;
+  // Logic Explanation:
+  //   - When this function returns true, it means the readonly.
+  //   - When this function returns false, it means allow editing the review.
+  const viewUserReviewProgramArea = () => {
```

Comment:
```suggestion
  const viewMode = (reviewerRole: string) => {
```

Reply:
along with changes down below to the function call

Reply:
or call the function `readOnly`

Reply:
thanks

> along with changes down below to the function call

thanks, I think the viewMode function name would be more intuitive